tools/xs-opam-ci.env

```diff
@@ -1,6 +1,6 @@
 export OCAML_VERSION="4.08"
 export OCAML_VERSION_FULL="4.08.1"
-export DISTRO="debian-unstable"
+export DISTRO="ocaml/opam2:debian-10-ocaml-4.08"
```

Comment:
The first part of the image name is filled by the script, only the distro should be exported:
```
from=${hub_user}/opam2:${DISTRO} ;;
```